### PR TITLE
Merge paper into dev

### PR DESCRIPTION
**Pull request by aprilsourz**
- Adds more styles to the app
- Upgrade to the newest version of ember-paper
- Add flash messages for user errors
- Scroll to bottom of chat box when a user sends a message
- refactor `getMessages()` method in services/profiles.js
   pulls out some of the code into status.js
-  Updates some of the button syntax to use an href attribute as is supported in ember-paper 1.0
-  Link to issue on ember-paper GitHub https://github.com/miguelcobain/ember-paper/issues/748

### CODE REVIEW

File: app/templates/components/recruiter-conversations/conversation.hbs [hunk 1]
```diff
@@ -11,15 +11,14 @@
...
     {{#link-to 'recruiter.view-conversation' conversation.id ~}}
       {{#paper-button}}View invitation{{/paper-button}}
     {{~/link-to}}
-    {{#paper-button~}}
-      <div {{action 'deleteConvo'}}> Delete </div>
-    {{~/paper-button}}
+    {{#paper-button onClick=(action "deleteConvo")}}Delete{{/paper-button}}
 
   </div>
 {{/paper-card}}
```
Comment (arlofeirman) [inline app/templates/components/recruiter-conversations/conversation.hbs]: Link to issue on ember-paper GitHub https://github.com/miguelcobain/ember-paper/issues/748

File: app/components/lunch-invites/invite/status.js [hunk 2]
```diff
@@ -2,18 +2,22 @@ import Ember from 'ember';
...
-  tagName: 'td',
-  classNames: ['grey'],
-  classNameBindings: ['unreadMessages'],
+  tagName: '',
   unreadMessages: null,
 
   onInit: function() {
     const id = this.get('conversation.id');
-    return this.get('profiles').getMessages(id)
+    return this.get('profiles').getMessagesForConversation(id)
```
Comment (spoonscen) [inline app/components/lunch-invites/invite/status.js:10]: You can clean this up a bit to make the code more declarative
```js
    const id = this.get('conversation.id');
    return this.get('profiles').getMessagesForConversation(id)
      .then((messages) => {
        const unreadMessages = messages.filter((message) => !message.read);
        if (unreadMessages.length) {
          this.set('unreadMessages', true);
        } else {
          this.set('unreadMessages', false);
        }
      })
```

File: app/components/recruiter-conversations/conversation/status.js [hunk 3]
```diff
@@ -8,11 +8,17 @@ export default Ember.Component.extend({
 
   onInit: function() {
     const id = this.get('conversation.id');
-    return this.get('profiles').getMessages(id)
+    return this.get('profiles').getMessagesForConversation(id)
+      .then((messages) => {
+        const unreadMessages = messages.filter((e) => e.read === false);
```
Comment (spoonscen) [inline app/components/recruiter-conversations/conversation/status.js:13]: same thing as above.
Using `e` as a variable in the filter is confusing. You should always give descriptive names :)
https://github.com/ryanmcdermott/clean-code-javascript#variables

File: app/templates/components/recruiter-conversations/conversation.hbs [hunk 4]
```diff
@@ -1,25 +1,26 @@
...
       {{recruiter-conversations/conversation/status conversation=conversation}}
     {{/title.media}}
   {{/card.title}}
 
   <div class="md-actions" layout-align="center">
-    <a target="_blank" href="{{conversation.engineer.website~}}">
-      {{#paper-button}}View their website{{/paper-button}}
-    </a>
+    {{!-- inconsistent syntax due to a problem with using a paper-button with an href attribute
+    filed an issue on the ember-paper github --}}
```
Comment (spoonscen) [inline app/templates/components/recruiter-conversations/conversation.hbs:18]: I thought you resolved this? Can you link to the issue here?

File: app/components/lunch-invites/invite/status.js (hunk 2 above)
Comment (spoonscen) [reply app/components/lunch-invites/invite/status.js:10]: By doing this you remove that extra `then` and you can clearly see what the code is trying to do.

File: app/templates/components/recruiter-conversations/conversation.hbs (hunk 4 above)
Comment (aprilsourz) [reply app/templates/components/recruiter-conversations/conversation.hbs:18]: The new button syntax in 1.0 works in other places but not here.
https://github.com/miguelcobain/ember-paper/issues/748

Comment (spoonscen) [reply app/templates/components/recruiter-conversations/conversation.hbs:18]: Cool! yeah put the issue link in the code comment!

**Pull request closed without merging**